photon-client/src/integTest/scala/com/linkedin/photon/ml/DriverTest.scala

```diff
@@ -29,7 +29,7 @@ import com.linkedin.photon.ml.PhotonOptionNames._
 import com.linkedin.photon.ml.TaskType.TaskType
 import com.linkedin.photon.ml.constants.MathConst
 import com.linkedin.photon.ml.diagnostics.DiagnosticMode
-import com.linkedin.photon.ml.io.{FieldNamesType, GLMSuite, InputFormatType}
+import com.linkedin.photon.ml.photon_io.{FieldNamesType, InputFormatType}
```

Comment:
Talked offline - we agreed to mark dirs/modules that are used only in "photon" but won't make it into "game" as `.deprecated`.